meta/main.yml

```diff
@@ -33,17 +33,13 @@ galaxy_info:
   #
   # platforms is a list of platforms, and each platform has a name and a list of versions.
   #
-  # platforms:
-  # - name: Fedora
-  #   versions:
-  #   - all
-  #   - 25
-  # - name: SomePlatform
-  #   versions:
-  #   - all
-  #   - 1.0
-  #   - 7
-  #   - 99.99
+  platforms:
+    - name: Centos
+      versions:
+        - all
+    - name: Ubuntu
+      versions:
+        - all
 
   galaxy_tags: []
```

Comment:
We should figure out what tags we might like when considering the use of Pacifica. Examples might be `data`, `archive`, `scientific`.

Reply:
Done